package.json

```diff
@@ -21,10 +31,10 @@
     "babel-cli": "^6.24.1",
     "babel-core": "^6.24.1",
     "babel-preset-es2015": "^6.24.1",
-    "babel-preset-stage-1": "^6.24.1"
+    "babel-preset-stage-1": "^6.24.1",
+    "postinstall-build": "^5.0.1"
```

Comment:
I added `postinstall-build` in the dev dependencies because `yarn install` gives me the following error:

```
$ postinstall-build lib
sh: postinstall-build: command not found
error Command failed with exit code 127.
```